src/mongocxx/database.cpp

```diff
@@ -82,7 +82,8 @@ cursor database::list_collections(bsoncxx::document::view_or_value filter) {
 
     return cursor(result);
 }
-stdx::string_view database::name() const {
+
+bsoncxx::stdx::string_view database::name() const {
```

Comment:
Doesn't look like this should have changed. We stopped saying bsoncxx::stdx in the mongocxx world a while ago.